examples/tasks/advanced/images/image_LICENSE.txt

```diff
@@ -3,7 +3,7 @@ licensed accordingly to the original license:
 
 Project                 License                   File
 ----------------------------------------------------------------------------
-Enthought               BSD 3-Clause              LICENSE.txt
+Enthought               Proprietary
```

Comment:
I concur.  We should only include files which have reasonably free licenses, so we'd either have to open the license file (which is problematic from the point of view of Enthought-as-a-business) or find something else to put there.